### ISSUE TITLE
Occultism: Capturing Demons Requirements

### ISSUE THREAD

**Issue by GhostOfTheSynthesis** [AllTheMods/ATM-9#1949]
### Modpack Version

0.2.60

### Describe the issue

Once I got to the Capturing Demons portion of the Occultism quest path, I started to get confused. The quest says I need four soul sand, a diamond, a copper ingot and a silver ingot (along with the book of binding) to complete the quest. However, using these items left a blank space in my bowl setup. Only upon joining the Occultism Discord I discovered you also need a gold ingot for this task. Please add it to the requirements.

tl;dr add gold ingot to the requirements in the Occultism Capturing Demons quest
